standard/statements.md

```diff
@@ -257,9 +260,12 @@ local_variable_declarator
 local_variable_initializer
     : expression
     | array_initializer
+    | stackalloc_initializer    // unsafe code support
     ;
 ```
 
+*stackalloc_initializer* is defined in [§23.9](unsafe-code.md#239-stack-allocation).
```

Comment:
Back in expressions.md these statements followed a "is only available in unsafe code" pattern, here in statements.md this is the second to follow an "is define in" pattern which doesn't directly mention "unsafe code". I suggested a change to the first occurrence (in *embedded_statement* to remove repetition (as suggested in expressions.md as well) which also changed it to the "is only available in unsafe code" pattern.

If there is no particular reason to change the pattern I'd suggest consistency and follow the pattern adopted in expressions.md. I won't flag them all, I've done the first one and this second would be:

> *stackalloc_initializer* ([§23.9](unsafe-code.md#239-stack-allocation)) is only available in unsafe code ([§23](unsafe-code.md#23-unsafe-code)).

If there is a good reason to change the pattern that keep it but I'd suggest removing any repetition as suggested above for *embedded_statement*.

All this is wordsmithing, the text is technically correct.